README.md

```diff
@@ -105,18 +110,25 @@ It's always possible to create a "root" (parentless) `Span`.
 
 ```go
     http.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
-        serverSpan, err := opentracing.GlobalTracer().Join(
-            "serverSpan",
+        var serverSpan opentracing.Span
+        appSpecificOperationName := ...
+		if wireMetadata, err := opentracing.GlobalTracer().Extract(
             opentracing.TextMap,
-            opentracing.HTTPHeaderTextMapCarrier(req.Header))
-
-        if err != nil {
-            // Create a root span if necessary
-            serverSpan = opentracing.StartTrace("serverSpan")
+            opentracing.HTTPHeaderTextMapCarrier(req.Header))); err != nil {
+            // Optionally record something about err here
+        }
+        if wireMetadata != nil {
+            // Create a span referring to the RPC client.
+            serverSpan = opentracing.StartSpan(
+                appSpecificOperationName,
+                opentracing.RefRPCClient.Point(wireMetadata))
+        } else {
+            // Create a root span.
+            serverSpan = opentracing.StartSpan(appSpecificOperationName)
         }
+
         var goCtx context.Context = ...
         goCtx, _ = opentracing.ContextWithSpan(goCtx, serverSpan)
```

Comment:
simplify: `ctx := opentracing.ContextWithSpan(ctx, serverSpan)` (it no longer returns 2 values)